src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background: url(../images/star.svg);
+    margin-right: 4px;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &--#{$stars-count} :nth-child(-n + #{$stars-count}) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &-reviews {
```

Comment:
The &-reviews seems to be a modifier for the .stars class but its usage isn't clear in this context. It should probably be &__reviews if it's an element of the .stars block according to BEM methodology.